packages/lib-react-components/src/Media/components/Audio/Audio.spec.jsx

```diff
@@ -21,18 +21,18 @@ describe('Audio', function () {
   describe('height and width', function () {
     it('should be set if specified', function () {
       render(<Audio height={200} width={270} src={audio} />)
-      const audioWrapper = document.querySelector('[class*="Audio__StyledBox"]')
+      const audioWrapper = document.querySelector('[class*="StyledBox"]')
```

Comment:
I'm not sure why this changed, but the Storybook UI looks unchanged and this test is still functional by simply looking for "StyledBox" in the class.